lib/taskana-core/src/main/java/pro/taskana/spi/routing/api/TaskRoutingProvider.java

```diff
@@ -14,10 +25,12 @@ public interface TaskRoutingProvider {
   void initialize(TaskanaEngine taskanaEngine);
```

Comment:
Please copy the JavaDoc from our newer SPIs which explain this method a lot better